server/auth/helpers.js

```diff
@@ -1,60 +1,33 @@
-import {merge} from 'lodash'
-
 import {connect} from 'src/db'
+import {User, UserAvatar, InviteCode} from 'src/server/services/dataService'
 
 const r = connect()
 
 export const defaultSuccessRedirect = '/'
 
-export function mergeUserInfo(user, userInfo) {
-  // don't overwrite these attributes
-  const {inviteCode, email, name, roles, createdAt, active, id} = user
-  return merge(user, userInfo, {inviteCode, email, name, roles, createdAt, active, id})
+export function mergeUserInfo(userInfo) {
```

Comment:
This doesn't seem to be merging anything anymore...and it doesn't really seem even necessary to have the function, either. Let's delete this function and move the logic into `server/auth/github.js` for now (which needs to be refactored eventually). 

There are two identical parts of code that use this function:
```js
user = await User
        .get(user.id)
        .updateWithTimestamp(mergeUserInfo(userInfo))
```

We can consolidate it into one function:
```js
function _updateUserGithubInfo(userId, userInfo) {
  return User
    .get(user.id)
    .updateWithTimestamp(mergeUserInfo(userInfo))
}
```